### PR TITLE
💄 Destructure globalShortcut module

### PR DESCRIPTION
**Pull request by stevekinney**
Minor change to conform with the ES6 destructuring in the latest version of the docs on master.

### CODE REVIEW

File: lib/browser/api/global-shortcut.js [hunk 1]
```diff
@@ -1,5 +1,3 @@
-var globalShortcut
-
-globalShortcut = process.atomBinding('global_shortcut').globalShortcut
+const {globalShortcut} = process.atomBinding('global_shortcut').globalShortcut
```
Comment (mnquintana) [inline lib/browser/api/global-shortcut.js]: I think this doesn't need the `.globalShortcut` at the end anymore

Comment (stevekinney) [inline lib/browser/api/global-shortcut.js]: Oh cool, I'll give it a shot and see what CI thinks!

File: lib/browser/api/global-shortcut.js [hunk 2]
```diff
@@ -1,5 +1 @@
-var globalShortcut
-
-globalShortcut = process.atomBinding('global_shortcut').globalShortcut
-
-module.exports = globalShortcut
+module.exports = process.atomBinding('global_shortcut')
```
Comment (zcbenz) [inline lib/browser/api/global-shortcut.js]: I think it should be `module.exports = process.atomBinding('global_shortcut').globalShortcut`?

Comment (stevekinney) [inline lib/browser/api/global-shortcut.js]: Okay, there was some discussion on an outdated diff that it wasn't needed. I put it back! 👍

Comment (zcbenz) [inline lib/browser/api/global-shortcut.js]: It is actually different case, you don't need that in `const {globalShortcut} = process.atomBinding('global_shortcut')`, but it is needed here.

**Pull request merged**